core/event_loop.go

```diff
@@ -31,6 +32,7 @@ func EventLoopWithInterrupt(agent *Agent, closeChan chan struct{}) error {
 
 	sigChan := make(chan os.Signal, 1)
 	signal.Notify(sigChan, os.Interrupt)
```

Comment:
Some of the examples I've using `signal.Notify()` I've seen use sys call.SIGINT instead of os.Interrupt. I suspect they are the same, but for consistency, I thought I'd mention this as well.

Reply:
It's essentially the same thing: https://golang.org/src/os/exec_posix.go?s=552:585#L18